package.json

```diff
@@ -1,38 +1,41 @@
 {
-  "name": "standupman",
-  "version": "1.0.0",
-  "description": "Seamless asynchronous updates for teams.",
-  "type": "module",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/standupman/standupman"
-  },
-  "main": "app.js",
-  "scripts": {
-    "start": "node --experimental-specifier-resolution=node app.js",
-    "dev": "nodemon --experimental-modules --es-module-specifier-resolution=node app.js",
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "openapi": "node openapi.js"
-  },
-  "author": "Fon E. Noel NFEBE <nomad@stacknerds.com>",
-  "license": "ISC",
-  "dependencies": {
-    "bcrypt": "^5.0.1",
-    "cors": "^2.8.5",
-    "dotenv": "^10.0.0",
-    "express": "^4.17.1",
-    "express-validator": "^6.12.0",
-    "jsonwebtoken": "^8.5.1",
-    "mongoose": "^5.12.13",
-    "passport": "^0.4.1",
-    "passport-jwt": "^4.0.0",
-    "swagger-jsdoc": "^6.1.0",
-    "swagger-ui-express": "^4.3.0"
-  },
-  "devDependencies": {
-    "eslint": "^8.12.0",
-    "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-plugin-import": "^2.25.4",
-    "nodemon": "^2.0.7"
+    "name": "standupman",
+    "version": "1.0.0",
+    "description": "Seamless asynchronous updates for teams.",
+    "type": "module",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/standupman/standupman"
+    },
+    "main": "app.js",
+    "scripts": {
+        "start": "node --experimental-specifier-resolution=node app.js",
+        "dev": "nodemon --experimental-modules --es-module-specifier-resolution=node app.js",
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "openapi": "node openapi.js",
+        "lint": "eslint .",
+        "lint:fix": "eslint --fix ."
+    },
+    "author": "Fon E. Noel NFEBE <nomad@stacknerds.com>",
+    "license": "ISC",
+    "dependencies": {
+        "bcrypt": "^5.0.1",
+        "cors": "^2.8.5",
+        "dotenv": "^10.0.0",
+        "express": "^4.17.1",
+        "express-validator": "^6.12.0",
+        "jsonwebtoken": "^8.5.1",
+        "mongoose": "^5.12.13",
+        "passport": "^0.4.1",
+        "passport-jwt": "^4.0.0",
+        "swagger-jsdoc": "^6.1.0",
+        "swagger-ui-express": "^4.3.0"
+    },
+    "devDependencies": {
+        "eslint": "^8.12.0",
+        "eslint-config-airbnb-base": "^15.0.0",
+        "eslint-plugin-import": "^2.25.4",
+        "nodemon": "^2.0.7"
+    }
   }
-}
+  
```

Comment:
Newline missing. You should not attempt fixing this, instead checkout to the original file and only add the new lines (17 to 17) which you are introducing.